docs/PersonalApplications.rst

```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
    Click on the notification stream, you will be taken to the detailed
    task pane in the Task Management application.
 
+.. _NewsNotifications :
+
+
+**News post in a space**
+
+You will receive the **News post in a space** notification when a user posts an article in a space of which you are member.
+
+	|image97|
+
+**News share in a space**
+
+You will receive the **News share in a space** notification when a user shares an article in a space of which you are member.
+
+	|image98|
+
+**My News shared**
+
+You will receive the **My News shared** notification when a user shares your posted article in another space.
+
+	|image99|
+
+**My News liked**
+
```

Comment:
Missing contents ?

Reply:
These notifications are not available yet. I only listed the list of upcoming notifications.

Reply:
:+1: